lib/pure/math.nim

```diff
@@ -114,6 +114,7 @@ const
                                            ## after the decimal point
                                            ## for Nim's ``float`` type.
   RadPerDeg = PI / 180.0 ## Number of radians per degree
+include mathconstants  # File containing only mathematical constants.
```

Comment:
wouldn't it be better to have that as it's own module?

Reply:
I dunno, maybe is better, what would be the name :grey_question: :thinking: